### DISCUSSION TITLE
I did a video about Smart Connections!

### DISCUSSION THREAD

**Discussion by rotello** [brianpetro/obsidian-smart-connections#178] [General]
I did a video (it s in Italian, but has english subs) about feeding your notes to ChatGPT. I used Obsidian and of Coz Smart Connection. It's not as Good as I wanted, but I hope people will discover this precious plugin.

I will post it on reddit and on Obsidian forum, too

https://www.youtube.com/watch?v=QZje9NaIqaM

**Comment by brianpetro** [comment]
I love this! Thanks @rotello
